doc/source/user_guide/copy_on_write.rst

```diff
@@ -94,32 +94,61 @@ rules. The returned array is set to non-writeable to protect against this behavi
 Creating a copy of this array allows modification. You can also make the array
 writeable again if you don't care about the pandas object anymore.
 
+See the section about :ref:`read-only NumPy arrays <copy_on_write_read_only_na>`
+for more details.
+
 **Only one pandas object is updated at once**
 
-The following code snippet updates both ``df`` and ``view`` without CoW:
+The following code snippet updates both ``df`` and ``subset`` without CoW:
 
 .. ipython:: python
 
     df = pd.DataFrame({"foo": [1, 2, 3], "bar": [4, 5, 6]})
     subset = df["foo"]
     subset.iloc[0] = 100
-    dfr
+    df
 
 This won't be possible anymore with CoW, since the CoW rules explicitly forbid this.
+This includes updating a single column as a :class:`Series` and relying on the change
+propagating back to the parent :class:`DataFrame`.
 This statement can be rewritten into a single statement with ``loc`` or ``iloc`` if
 this behavior is necessary. :meth:`DataFrame.where` is another suitable alternative
 for this case.
 
+Updating a column selected from a :class:`DataFrame` with an inplace method will
+also not work anymore.
+
+.. ipython:: python
+    :okwarning:
+
+    df = pd.DataFrame({"foo": [1, 2, 3], "bar": [4, 5, 6]})
+    df["foo"].replace(1, 5, inplace=True)
+    df
+
+This is another form of chained assignment. This can generally be rewritten in 2
+different forms:
+
+.. ipython:: python
+
+    df = pd.DataFrame({"foo": [1, 2, 3], "bar": [4, 5, 6]})
+    df.replace({"foo": 1}, {"foo": 5}, inplace=True)
```

Comment:
```suggestion
    df.replace({"foo": {1: 5}}, inplace=True)
```

Both work, personally I find this version a bit clearer

Reply:
I don't really care either way :)